src/components/FicheFilm/FicheFilm.jsx

```diff
@@ -118,19 +169,30 @@ export default function FicheFilm() {
               return <p>{genre.name}</p>;
             })}
           </div>
-          <p className="overview">{overviews}</p>
+          <p className="overview">{overview}</p>
         </div>
       </div>
+
       <div className="buttons">
-        <button type="button" className="buttonAdd">
-          +
-        </button>
-        <a href={`https://www.youtube.com/embed/${trailers.key}`}>
-          <div className="buttonTrailer" alt="trailer">
+        {watchlist.find((movie) => movie.Id === id) ? (
+          <button type="button" onClick={deleteToWatch} className="buttonAdd">
+            <span className="buttonText+or-">Removie!</span>
+          </button>
+        ) : (
+          <button type="button" onClick={addToWatch} className="buttonAdd">
+            <span className="buttonText+or-">Add to Watch!</span>
+          </button>
+        )}
+
+        <a href={`https://www.youtube.com/embed/${trailer.key}`}>
+          <div className="buttonTrailer" alt="Trailer">
+
+
```

Comment:
Useless empty lines + Prettier wouldn't have let this pass, what happened ?

Reply:
I added corrections!